Sources/StreamVideo/WebRTC/SfuMiddleware.swift

```diff
@@ -138,6 +140,17 @@ class SfuMiddleware: EventMiddleware {
         log.debug("Participant \(participant.name) joined the call")
     }
     
+    private func handleParticipantUpdated(_ event: Stream_Video_Sfu_Event_ParticipantUpdated) async {
+        let callParticipants = await state.callParticipants
+        let existing = callParticipants[event.participant.sessionID]
+        let participant = event.participant.toCallParticipant(
```

Comment:
shall we avoid any action if existing doesn't exist? (which should never be the case for an update)

Reply:
In this case, if it doesn't exist, it would be added to the list. This is a fallback, if for some reason we haven't received the participant joined event (although it shouldn't happen). I'm also fine with no action, let me know what you think.

Reply:
Going back to this one - actually it can happen that we don't have the participant. The join request returns max 250 participants, so not all can be loaded. If a participant in the rest of the chunk is updated, we won't have it.